core/models/officeblock.py

```diff
@@ -21,3 +21,23 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+class OfficeFloor(models.Model):
+    number = models.PositiveIntegerField(blank=False,
+                                         null=False, unique=True)
+    block = models.ForeignKey(OfficeBlock, on_delete=models.PROTECT)
+
+    def save(self, *args, **kwargs):
+        try:
+            self.full_clean()
+        except Exception as e:
+            raise ValidationError(e)
+        super().save(*args, **kwargs)
+
+    class Meta:
+        verbose_name = 'Floor'
```

Comment:
Update the verbose_name to `Office Floor` to be consistent on admin site.

<img width="273" alt="screen shot 2018-07-26 at 13 08 08" src="https://user-images.githubusercontent.com/10899666/43256302-032e239c-90d5-11e8-9448-ee08904c35ca.png">